app/components/forms/wizard/other-details-step.js

```diff
@@ -39,6 +39,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return !!this.data.event.codeOfConduct;
   }),
 
+  ticketsPresent: computed('data.event.tickets.@each', function() {
+    return this.data.event.tickets.length > 0;
+  }),
```

Comment:
This is present in all 3 components

Reply:
can I put it in event-wizard? Will it work from there for all this components?